frontend/packages/pipelines-plugin/src/utils/__tests__/pipeline-filter-reducer.spec.ts

```diff
@@ -34,7 +34,7 @@ const mockPipelineRuns = [
   { status: { conditions: [{ status: 'Unknown', type: 'Succeeded' }] } },
   {
     status: {
-      conditions: [{ type: 'Succeeded', status: 'Unknown', reason: 'PipelineRunCancelled' }],
+      conditions: [{ type: 'Succeeded', status: 'Unknown', reason: 'StoppedRunFinally' }],
```

Comment:
Add a test case where both the new statuses `StoppedRunFinally` and `CancelledRunFinally` are considered as `Cancelled` state.

Reply:
Added tests.